mobile/src/navigation/app/components/drawer-content/drawer-content.tsx

```diff
@@ -0,0 +1,93 @@
+import { DrawerContentComponentProps } from '@react-navigation/drawer';
+import React, { FC } from 'react';
+
+import billing from '~/assets/icons/billing.svg';
+import book from '~/assets/icons/book.svg';
+import education from '~/assets/icons/education.svg';
+import home from '~/assets/icons/home.svg';
+import mentors from '~/assets/icons/mentors.svg';
+import settings from '~/assets/icons/settings.svg';
+import { AppScreenName } from '~/common/enums/enums';
+import { Button, Image, ScrollView, View } from '~/components/common/common';
+import { CustomDrawerList } from '~/navigation/app/components/drawer-list/drawer-list';
+
+import { styles } from './styles';
+
+const routes = [
```

Comment:
Move this to drawer-content/common/contants/constants.ts
```suggestion
const NAVIGATION_ITEMS = [
```